client/layout/global-sidebar/main.jsx

```diff
@@ -71,7 +73,7 @@ const GlobalSidebar = ( {
 	}
 
 	const { requireBackLink, siteTitle, backLinkHref, ...sidebarProps } = props;
-	const sidebarBackLinkHref = backLinkHref || previousLink || '/sites';
+	const sidebarBackLinkHref = urlFrom || backLinkHref || previousLink || '/sites';
```

Comment:
Should we validate `urlFrom` to ensure it's safe, as it can be set via URL by anyone?

Reply:
Good point, I can add a check for relative paths only 🙂